server/helpers/validation.ts

```diff
@@ -3,8 +3,11 @@ import { validate, ValidationSchema } from 'class-validator';
 import { Request, Response, NextFunction } from 'express';
 import { Post } from '../entities/post';
 import Boom from '@hapi/boom';
+import { Reply } from '../entities/reply';
 // As the validation will be put to use on all the classes the structure of this file will need some refactoring.
 
+const validationError : Boom = Boom.badRequest("Validation error");
```

Comment:
You don't have to type the variable explicitly. The type is infered

Reply:
Hmm, I suppose coming from the Java world where you always have to declare the type upfront made me assume it's just good practice to do so in TypeScript as well? This being said though, I'm fairly certain that in situations like this where TypeScript doesn't force the type declaration from the user, many of my own variables are missing them too! 

I'll collect those ": Boom"s out and try to see if I spot similar mistakes!

Reply:
TypeScript is not so verbose compared to Java. That is why in TS it is good practice to leave typing out if the type is infered correctly like in this case